### PR TITLE
unixPB: Add ssh client to CentOS common packages

### LINKED ISSUE #1989: Docker images used for building should be able to clone over ssh
Some compiles run in Docker (eg xlinux centos6_build_image). I have a requirement to clone over ssh (git@github) but the container is missing ssh (possibly other deps). Can we add all the dependencies required to all the build images? I at least need them for all the OpenJ9 platforms which at the moment is xlinux, aarch linux and riscv linux.
centos6_build_image
centos7_build_image

### PR DESCRIPTION
**Pull request by AdamBrousseau**
Fixes #1989

Signed-off-by: Adam Brousseau <adam.brousseau88@gmail.com>

##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md#commit-messages)
- [ ] [FAQ.md](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md) updated if appropriate
- [x] other documentation is changed or added (if applicable)
- [x] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
Build by hand on my laptop and it appears to solve 1989. Marking for review.

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml [hunk 1]
```diff
@@ -45,6 +45,7 @@ Build_Tool_Packages:
   - perl-GD
   - perl-libwww-perl
   - perl-Time-HiRes
+  - openssh-clients
```
Comment (sxa) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml:48]: Can you add a comment saying `Required by IBM to clone over ssh` or words to that effect so we remember why it's there (I can't easily get the comment lined up correctly by making a "suggested change" in the github online editor so I'm not going to try!)

**Review by sxa — CHANGES_REQUESTED** [review]

**Comment by sxa** [comment]
I'm just going to quickly run this through VagrantPlaybookCheck to make sure the package name is ok on other CentOS versions - sholdn't be more htan 15 minutes or so before it gets to the important part: https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/1105/

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml [hunk 2]
```diff
@@ -45,6 +45,7 @@ Build_Tool_Packages:
   - perl-GD
   - perl-libwww-perl
   - perl-Time-HiRes
+  - openssh-clients               # IBM: cloning over SSH
```
Comment (sxa) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml:48]: Just realised - this is also not in the right place alphabetically - sorry to be picky but can we move it to just before `openssl-devel` please?

**Review by sxa — CHANGES_REQUESTED** [review]

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml (hunk 2 above)
Comment (AdamBrousseau) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml:48]: Right, good catch. I had it there as `ssh` before I found the right package name.

**Review by sxa — APPROVED** [review]
LGTM - Thanks for the changes - The linter error is not related to this change

**Comment by AdamBrousseau** [comment]
Will the images be regenerated automatically or is there another process someone needs to initiate?

**Comment by sxa** [comment]
It should be automatic

**Comment by AdamBrousseau** [comment]
For completeness, or the next time I try to find it
https://ci.adoptopenjdk.net/job/openjdk_build_docker_multiarch/
Launched daily at 5:07pm so this change would be included in tomorrow's build number 253.

**Comment by AdamBrousseau** [comment]
I'm not convinced that is the right build for the build_images
@gdams can you confirm how these are updated?
https://hub.docker.com/r/adoptopenjdk/centos6_build_image
https://hub.docker.com/r/adoptopenjdk/centos7_build_image

**Pull request merged**